_config.yml

```diff
@@ -5,9 +5,9 @@
 # DO NOT CHANGE THE LINE NUMBERS HERE without changing .circleci/circle_urls.sh
 # If you are building a simple GitHub user page (https://username.github.io) then use these settings:
 url: "https://us-rse.org"
-baseurl: ""
-title-img: /assets/img/logo_600px.png
-twitter-img: https://us-rse.org/assets/img/logo_600px.png
+baseurl: ""  # set via .circleci/circle_urls.sh -- set it there
```

Comment:
This isn't entirely correct and could be confusing - a baseurl is a [jekyll known thing](https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/) that determines what the site base name is. The reason we change it in the continuous integration folder, with a script called circle_urls.sh, is that we are tweaking it last minute so that it builds with the circle basename (which is specific to the build, like `https://233-120906408-gh.circle-artifacts.com/0/usrse.github.io`) and not the default that won't work. The CI files and the config.yml should be considered separately - if you add a comment or note anywhere, add it to the circle_urls.sh, but probably not best here. A person wanting to change the baseurl of the site would not need or want to set it in the circle_urls.sh.

Reply:
I think there needs to be some comment in _config.yml.  What would you suggest putting besides what's there?  Maybe a note that for testing, circle_urls.sh needs to be updated as well?

Reply:
In _config.yml I would link to the [official docs](https://jekyllrb.com/docs/configuration/options/#serve-command-options) to explain base url, and then put an **important** the index of this line is used in CircleCI testing, change in circle_urls.sh if modified here. See circle_urls.sh for details.

Reply:
In the latest comment (below), I have `# for testing, also check .circleci/circle_urls.sh `.  Sound ok?